.github/workflows/test.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '0 */12 * * *'
+
+env:
+  GO_VERSION: 1.15.1
+  K8S_VERSION: v1.19.0
+  KIND_CLUSTER_NAME: ovn
+  KIND_INSTALL_INGRESS: true
+  KIND_ALLOW_SYSTEM_WRITES: true
```

Comment:
Are these VARS needed?
  K8S_VERSION
  KIND_CLUSTER_NAME
  KIND_INSTALL_INGRESS
  KIND_ALLOW_SYSTEM_WRITES

Reply:
It has been tested. Since it is run on both push and pull request. It was tested in my fork by pushing to master.

Reply:
The VARS are not used in this PR so I can remove them to reduce confusion.